static/js/dropdown_list_widget.js

```diff
@@ -8,6 +8,7 @@ import * as ListWidget from "./list_widget";
 export const DropdownListWidget = function ({
     widget_name,
     data,
+    retain_current_item = true,
```

Comment:
Yea, `include_current_item` does sound better.
What about `exclude_current_item`? Since adding a default parameter of `false` and later setting that to `true` within it's use case feels more natural. 
However, kept `include_current_item` for now. As discussed earlier, Will open up a thread as a follow-up work regarding the same.